src/main/java/dk/kb/labsapi/ImageExport.java

```diff
@@ -250,12 +263,16 @@ public void exportFullpages(String query, Integer startYear, Integer endYear, In
         Stream<SolrDocument> docs = streamSolr(finalQuery, max);
         // Create metadata file, that has to be added to output zip
         Map<String, Object> metadataMap = makeMetadataMap(query, startYear, endYear);
+
+        Set<String> fields = new HashSet<>(Arrays.asList("pageUUID", "familyId", "lplace", "fulltext_org"));
+        StreamingOutput csvStream = SolrExport.getInstance().export(query, fields,(long) max, SolrExport.STRUCTURE.DEFAULT , SolrExport.EXPORT_FORMAT.csv );
```

Comment:
This is quite tricky: We want the CSV for _all_ the articles on the exported pages but no more. The current query just return the first `max` articles, in ranking order.

Reply:
Sorry but your implementation will not work. I'll start a new review as this comment is about the new code.